doc/modules/fuzzy_check.md

```diff
@@ -11,39 +11,122 @@ is responsible for teaching fuzzy storage with message patterns.
+		min_bytes = ...; //int: Override module default min bytes for this rule
+		read_only = ...; //boolean: If true then never try to train this fuzzy storage
+		servers = "..."; //string: List of servers to check (or train)
+		short_text_direct_hash = ...; //boolean: Use direct hash for short texts
+		skip_hashes = "..."; //string: Whitelisted hashes map
+		skip_unknown = ...; //boolean: If true then ignores unknown flags and does not add the default fuzzy symbol
+		symbol = "..."; //string: Default symbol for rule (if no flags defined or matched)
+	}
+}
+~~~
+
 ## Module configuration
 
-The fuzzy check module has several global options and allows to specify multiple match
-storages. Global options include:
+The ```fuzzy_check``` module has several global options, including:
 
-- `symbol`: default symbol to insert (if no flags match)
-- `min_length`: minimum length of text parts in words to perform fuzzy check (default - check all text parts)
 - `min_bytes`: minimum length of attachments and images in bytes to check them in fuzzy storage
+- `min_height`: minimum pixel height of images to be checked
+- `min_length`: minimum length of text parts in words to perform fuzzy check (default - check all text parts)
+- `min_width`: minimum pixel width of images to be checked
+- `retransmits`: maximum retransmissions before giving up
+- `symbol`: default symbol to insert (if no flags match)
+- `timeout`: timeout to wait for a reply, e.g. 1s, 2m, 5h
 - `whitelist`: IPs in this list bypass all fuzzy checks
-- `timeout`: timeout to wait for a reply
 
-Fuzzy rules are defined as a set of `rule` definitions. Each `rule` must have a `servers`
-list to check or learn, and a set of flags and optional parameters. Here is an example:
+e.g.
+~~~ucl
+# local.d/fuzzy_check.conf
+# the following are defaults in 1.9.4
+fuzzy_check {
+    min_bytes = 1k; # Since small parts and small attachments cause too many FP
+    timeout = 2s;
+    retransmits = 1;
+    ...
+    rule {...}
+}
+~~~
+
+
+A fuzzy `rule` is defined as a set of `rule` definitions. Each `rule` must have a `servers`
+list to check or learn, and a set of flags and optional parameters. 
+
+The `servers` parameter can have the round-robin parameter to alternately try each entry, e.g. using a list:
+```
+        servers = "round-robin:fuzzy1.rspamd.com:11335,fuzzy2.rspamd.com:11335";
+```
+
+Usable parameters include:
+
+- `algorithm`: rule hashing algo; one of: `fasthash` (or just `fast`), `mumhash`, `siphash` (or `old`) or `xxhash`
+- `encryption_key`: Base32 value public key to access this rule
+- `fuzzy_map`: Map of SYMBOL -> data for flags configuration
+- `fuzzy_key`: Base32 value for the hashing key (for private storages).
+- `learn_condition`: An Lua script that returns a boolean function to check whether this task
+	should be considered when training fuzzy storage
+- `max_score`: float value for this rule's max possible score
```

Comment:
Max score is not the max possible score. I think it is better to write that it is described below.

Reply:
As I said, much of this text originates from the source code - these I think were LUA help strings....  will go with your version.

Reply:
now?